Listener/RestListener.php

```diff
@@ -162,7 +162,7 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
         if ($cacheStrategy !== null && $cacheStrategy instanceof ResponseAwareCacheStrategy) {
             $cacheStrategy->setResponse($response);
         } else {
-            $response->setPrivate();
+            $response->headers->set('Cache-control', 'no-store, no-cache, must-revalidate');
```

Comment:
First you need to replicate the logic inside `Response::setPrivate` because now it is possible to have conflicting `Cache-Control: no-store, store`. Then please use `HeaderBag::addCacheControlDirective`